src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -300,6 +300,19 @@ export default class AircraftCommander {
         return aircraft.pilot.initiateHoldingPattern(fixName, holdParameters);
     }
 
+    /**
+     * Exit holding pattern and resume navigation
+     *
+     * @for AircraftCommander
+     * @method runExitHold
+     * @param aircraft {AircraftModel}
+     * @param data {array}
```

Comment:
this param is not actually in the method definition, remove from docblock

Reply:
done